app/controllers/admin/redeem_requests_controller.rb

```diff
@@ -32,10 +30,27 @@ def destroy
     redirect_to admin_redeem_requests_path
   end
 
+  def download
+    csv_string = CSV.generate do |csv|
+      csv << ["User","Gift Shop", "Store","Points","Cost","Date","Coupon Code","Address","Status"]
+      @redeem_requests.each do |redeem_request|
+        csv << [redeem_request.user.email,redeem_request.retailer,redeem_request.store,redeem_request.points,redeem_request.points/10,redeem_request.updated_at.strftime(fmt='%F %T'),redeem_request.coupon_code,redeem_request.address,redeem_request.status]
+      end
+    end         
+   send_data csv_string,
```

Comment:
@amitk You need not place the parameters on seperate lines. Also, use 'type: ...' syntax rather than ':type => ...'.